docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,46 @@
+# Inleiding
+Een verzoek is een aanvraag of opdracht aan de gemeente voor de levering van een product of dienst. Verzoeken vormen dus de schakel tussen de klant, producten en diensten van de gemeente zoals in de PDC staan, en het zaakgericht werken voor de afhandeling. Dit biedt een aantal voordelen, onder andere op het gebied van herkenbaarheid voor de klant, die niet meer wordt geconfronteerd met voor hem onherkenbare concepten uit de gemeentelijke bedrijfsvoering. Voor gemeenten ontstaat bovendien meer flexibiliteit: een aanvraag van een klant leidt niet langer vanzelfsprekend tot een zaak, en één of meerdere verzoeken kunnen worden gerelateerd aan één of meerdere zaken.
```

Comment:
Ik begrijp dat verzoeken de schakel vormen tussen de klant en zaakgericht werken maar waarom geldt dat ook voor producten en diensten en zaakgericht werken? Ik heb het beeld dat het zaakgericht werken de invulling van een product of dienst is.

Reply:
Verzoeken vormen de schakel tussen klant en bedrijfsvoering (die zaakgericht kan zijn). Producten en diensten zijn de definities van wat je kunt aanvragen. Verzoek en daaraan gekoppelde zaken of andere activiteiten realiseren producten/diensten.